python/examples/minimal_build/README.md

```diff
@@ -24,38 +24,42 @@ an eye toward maximum portability across different Linux distributions. This
 may help for contributors debugging build issues caused by their local
 environments.
 
-## Fedora 31
+## Ubuntu 18.04
 
 Build image:
 
 ```
-docker build -t arrow_fedora_minimal -f Dockerfile.fedora
+docker build -t arrow_ubuntu_minimal -f Dockerfile.ubuntu
```

Comment:
```suggestion
docker build -t arrow_ubuntu_minimal -f Dockerfile.ubuntu .
```

Reply:
(without this change, the `docker` command fails:
```
"docker build" requires exactly 1 argument.
See 'docker build --help'.
```
)

Reply:
Added though these lines are originally from Wes